tests/database_metrics/test_monitor.py

```diff
@@ -18,33 +18,32 @@ def test_measure_database(self, mock_metrics, mock_parser, mock_clear):
         # mock_container.exec_run.return_value = None
         # mock_docker_client = MagicMock()
         # mock_docker_client.containers.run.return_value = mock_container
-
         mock_metrics.return_value = None
         mock_clear.return_value = None
         mock_parser.side_effect = [
-            {"table_rows": 1, "db_disk_usage_mb": 2., "runtime": 3, "memory_usage_mb": 4},
-            {"table_rows": 5, "db_disk_usage_mb": 6., "runtime": 7, "memory_usage_mb": 8},
-            {"table_rows": 9, "db_disk_usage_mb": 10., "runtime": 11, "memory_usage_mb": 12},
-            {"table_rows": 2, "db_disk_usage_mb": 4., "runtime": 6, "memory_usage_mb": 8},
-            {"table_rows": 1, "db_disk_usage_mb": 3., "runtime": 5, "memory_usage_mb": 7},
-            {"table_rows": 3, "db_disk_usage_mb": 6., "runtime": 9, "memory_usage_mb": 12},
+            {"a": 1, "b": 2., "c": 3, "d": 4},
+            {"a": 5, "b": 6., "c": 7, "d": 8},
+            {"a": 9, "b": 10., "c": 11, "d": 12},
+            {"a": 2, "b": 4., "c": 6, "d": 8},
+            {"a": 1, "b": 3., "c": 5, "d": 7},
+            {"a": 3, "b": 6., "c": 9, "d": 12},
```

Comment:
What's this about?

Reply:
The function doesn't actually care what the returned dict content is since it just appends them to the output list, and so instead of 
i) updating the test mocks to look like the returned mocks (which could get quite large, like in this PR where we add 2 more fields), or 
ii) leaving in the existing mocks (which would pass fine) and potentially confusing someone as to why they don't line up with what parse_metrics() actually returns, 
I just made them nonsense values to show that it just adds the parsed values, whatever they may be, to the output list.

I'm not sure it's the best idea, and could certainly make it more clear by making them single-key dicts.

Reply:
Yep. Recommend something like
```
{'toy': 'output'}
```

for clarity

Reply:
good idea, updated